server/activity/activity.model.js

```diff
@@ -229,9 +229,9 @@ class Activity extends Model {
   }
 }
 
-function removeAll(Model, where = {}, soft = false) {
+function removeAll(Model, where = {}, soft = false, transaction) {
```

Comment:
Um, I'm entering nitpick mode right now but my OCD begs for joining soft and transaction into optional options arg. 🙃